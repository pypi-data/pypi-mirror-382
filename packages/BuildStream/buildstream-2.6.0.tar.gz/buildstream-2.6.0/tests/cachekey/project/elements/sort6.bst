kind: stack

depends:
- elements/sort7.bst
