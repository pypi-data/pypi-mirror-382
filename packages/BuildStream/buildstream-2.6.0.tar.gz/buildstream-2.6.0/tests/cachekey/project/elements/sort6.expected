292431a52a4f31d9e45ff46e124e74a6306424fb7175dca4166d876d3fd1ae62
